### PR TITLE
Release for v0.7.7

### PR DESCRIPTION
**Pull request by lzchen**
Minor release. Consists of mostly `azure` related changes.

Trace span attribute changes: [#827]
Log Exporter custom dimension API changes: [#837]
Local storage + retry for metrics exporter: [#845]
Fixed rate sampling for log exporter: [#848]
Telemetry processors for exporters: [#851]

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+## 0.7.7
+Released 2020-02-03
```
Comment (c24t) [inline CHANGELOG.md]:
```suggestion
Released 2020-02-03

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+- Update `azure` module
```
Comment (c24t) [inline CHANGELOG.md]:
```suggestion
- Updated `azure` module
```

To be consistent with the others

**Comment by lzchen** [comment]
> Do we need to update the minimum required `opencensus` version for `opencensus-ext-httplib` and `opencensus-ext-requests`?
> 
> Look good otherwise.

Correct if if I'm wrong, but don't we need to update the minimum required version of `opencensus` only if there were new changes in `opencensus` that `opencensus-ext-httplib` and `opencensus-ext-requests` would rely on?

**Comment by c24t** [comment]
You're right, I see that these changes just add some new span attributes in the ext packages.

You might want to change the message in the changelog though. "Support span attribute changes in `opencensus>=0.7.7`" makes it sound like the ext libraries were changed to support attribute changes in the core library.

**Pull request merged**